_quickstarts/deploy.markdown

```diff
@@ -0,0 +1,114 @@
+---
+htmlid: deploy
+tab_title: Deployment
+---
+Kroxylicious is a Java application based on [Netty](https://netty.io/), which means it will run anywhere you can run a JVM. (That's a lot of places!)
+To help you get started with Kroxylicious, we've created this quick setup guide.
+
+<br />
+
+# Getting started
+
+### Prerequisites
+
+#### Java
+
+To get started deploying Kroxylicious, you will need to install a Java Runtime Environment (JRE) with minimum version 17. This does not come included with Kroxylicious.
+
+Some operating systems come with a JRE already installed. You can check what Java version you have installed by running the following command:
+
+```shell
+java -version
+```
+
+{% capture jre_note %}
+If you get an error or the command doesn't return anything, you may not have a JRE installed, or your JRE may not have been correctly added to your system's `PATH` variable.
+{% endcapture %}
+
+{% include bs-alert.html type="primary" icon="info-circle-fill" content=jre_note %}
+
+#### Apache Kafka®
+
+You will also need a running Apache Kafka® cluster to point Kroxylicious at. The Strimzi project has an [easy-to-follow quickstart](https://strimzi.io/quickstarts/) for setting up a cluster on your local machine using Minikube, or alternatively you can follow the official Apache Kafka® [quickstart](https://kafka.apache.org/documentation/#quickstart) to set up a local bare metal cluster.
```

Comment:
I had problems with the rendering of the registered sign.  I had to use the entity ref.

Reply:
how about?
```suggestion
You will also need a running Apache Kafka® cluster to be proxied by Kroxylicious. The Strimzi project has an [easy-to-follow quickstart](https://strimzi.io/quickstarts/) for setting up a cluster on your local machine using Minikube, or alternatively you can follow the official Apache Kafka® [quickstart](https://kafka.apache.org/documentation/#quickstart) to set up a local bare metal cluster.
```

Reply:
The strimzi quickstart gives you a kubernetes cluster within Kubenetes.  You need to do extra stuff to expose the cluster to the host network (nodeports, advertised listeners etc), otherwise the Kroxylicious instance running locally won't be able to connect to it.

Reply:
> I had problems with the rendering of the registered sign. I had to use the entity ref.

Fixed!

> how about?

I ended up going with "You will also need a running Apache Kafka&#174; cluster for Kroxylicious to proxy." as I thought it read better in the context of the rest of the quickstart.

> The strimzi quickstart gives you a kubernetes cluster within Kubenetes. You need to do extra stuff to expose the cluster to the host network (nodeports, advertised listeners etc), otherwise the Kroxylicious instance running locally won't be able to connect to it.

I had forgotten about that, good point. In the interest of keeping things simple I've just removed the references to the Strimzi quickstart entirely. Maybe we can have a blog post at some point in the future about setting up Kroxylicious with a Strimzi cluster.